src/parser/metrics/RealLinesOfCode.ts

```diff
@@ -81,6 +99,20 @@ export class RealLinesOfCode implements Metric {
         };
     }
 
+    /**
+     * Checks whether the node should count as a comment in python. Special handling for python multiline comments.
+     * @param node The node to check.
+     * @return Whether the node is a comment.
+     */
+    isPythonComment(node: Parser.SyntaxNode) {
+        return (
+            this.commentStatementsSet.has(node.type) ||
+            (node.type === "expression_statement" &&
+                node.childCount === 1 &&
+                node.child(0)?.type === "string")
+        );
+    }
+
```

Comment:
I think the comment does not add value. So let's omit if to have a better maintainability.
```suggestion
    isPythonComment(node: Parser.SyntaxNode) {
        return
            this.commentStatementsSet.has(node.type) ||
            (node.type === "expression_statement" &&
                node.childCount === 1 &&
                node.child(0)?.type === "string");
    }
```

Reply:
I think we should mention what the special treatment for Python exactly does. But it does not have to be a fully-fledged JSDoc-Comment, I agree.

Reply:
Hm okay. Then we should maybe do it like this and name the function for what it is actually doing. By that, I think a comment is not needed anymore. Maybe a comment is needed at the place where this method is called.
```suggestion
    isPythonMultiLineComment(node: Parser.SyntaxNode) {
        return (
            this.commentStatementsSet.has(node.type) ||
            (node.type === "expression_statement" &&
                node.childCount === 1 &&
                node.child(0)?.type === "string")
        );
    }

```

Reply:
Well, it does also check whether it is an ordinary comment (`this.commentStatementsSet.has(node.type)`) as python does also include those, so this name would be misleading. Maybe the comment was misleading as well in that case.

Reply:
I replaced the JSDoc-comment with a more clear documentation of what the special handling actually does and added a comment to the usage side as well.

Reply:
I have now done the following, as "isPythonSingleLineComment" is identical to how comments are detected in all other languages:

```typescript
isPythonComment(node: Parser.SyntaxNode) {
        return this.isComment(node) || this.isPythonMultilineComment(node);
}
```